.github/workflows/e2e-cypress-alt.yml

```diff
@@ -0,0 +1,96 @@
+name: E2E Tests Alternate Preferences
+
+on: pull_request
+
+jobs:
+  setup-env:
+    name: 'Load ENV Vars'
+    uses: ./.github/workflows/setup-env.yml
+    secrets: inherit
+
+  e2e-tests:
+    runs-on: ubuntu-latest
+    needs: [setup-env]
+
+    services:
+      elasticsearch:
+        image: docker.elastic.co/elasticsearch/elasticsearch:8.13.4
+        options: --health-cmd="curl --silent --fail localhost:9200/_cluster/health || exit 1" --health-interval=10s --health-timeout=5s --health-retries=5
+        ports:
+          - 9200:9200
+          - 9300:9300
+        env:
+          discovery.type: single-node
+          xpack.security.enabled: false
+
+      redis:
+        image: redis
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 20s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - '${{vars.REDIS_PORT}}:${{vars.REDIS_PORT}}'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Configure sysctl limits
+        run: |
+          sudo swapoff -a
+          sudo sysctl -w vm.swappiness=1
+          sudo sysctl -w fs.file-max=262144
+          sudo sysctl -w vm.max_map_count=262144
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/*'
+          check-latest: true
+      - run: npm ci
+
+      - run: npm run copyAltTestPreferences
+
+      - name: 'Create env file'
+        run: |
+          ENV_FILE_PATH=./apps/server/.env
+          touch ${ENV_FILE_PATH}
+
+          # Vars
+          echo -e "${{ needs.setup-env.outputs.env_vars }}" >> ${ENV_FILE_PATH}
+          echo RESOURCEVERSION="" >> ${ENV_FILE_PATH}
+
+          # Secrets (can't load these from another job, due to GH security features)
+          echo UCPAUTHCLIENTID=${{ secrets.UCP_CLIENT_ID }} >> ${ENV_FILE_PATH}
+          echo UCPAUTHCLIENTSECRET=${{ secrets.UCP_CLIENT_SECRET }} >> ${ENV_FILE_PATH}
+          echo UCPAUTHENCRYPTIONKEY=${{ secrets.UCP_ENCRYPTION_KEY }} >> ${ENV_FILE_PATH}
+
+          echo SOPHTRONAPIUSERID=${{ secrets.SOPHTRON_API_USER_ID }} >> ${ENV_FILE_PATH}
+          echo SOPHTRONAPIUSERSECRET=${{ secrets.SOPHTRON_API_USER_SECRET }} >> ${ENV_FILE_PATH}
+
+          echo MXCLIENTID=${{ secrets.MX_CLIENT_ID }} >> ${ENV_FILE_PATH}
+          echo MXAPISECRET=${{ secrets.MX_API_SECRET }} >> ${ENV_FILE_PATH}
+
+          echo FINICITYPARTNERID=${{ secrets.FINICITY_PARTNER_ID }} >> ${ENV_FILE_PATH}
+          echo FINICITYSECRET=${{ secrets.FINICITY_SECRET }} >> ${ENV_FILE_PATH}
+          echo FINICITYAPPKEY=${{ secrets.FINICITY_APP_KEY }} >> ${ENV_FILE_PATH}
+
+          cat ${ENV_FILE_PATH}
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v6
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          spec: apps/server/cypress/e2e/suite2/*.cy.ts
```

Comment:
Instead of using spec we should probably define a second cypress config that points to the tests we want to run with this.

The way it is right now if I do cypress:run locally, then the tests will fail because it's running a test it probably shouldn't be.

Reply:
It seems like we will need more commands for locally running the tests.

cypress:open
cypress:open:alternate

cypress:run
cypress:run:alternate